tests/happy-path.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test, expect, _electron as electron } from '@playwright/test';
+import { mkdtemp, rm } from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import { sep, join } from 'node:path';
+
+let installDir: string;
+
+
+test.beforeAll(async () => {
```

Comment:
I think we probably want this to be a `beforeEach`, rather than a `beforeAll`, since each individual test should get its own little temporary fiefdom.

Reply:
Might not be needed at all: https://github.com/drupal/cms-launcher/pull/114#issuecomment-3169662145